File: python/average.py
def average(arr) -> int:
    if len(arr) <= 0:
        raise ValueError("invalid array")
    
    sum:int = 0

    for i in arr:
        sum += i

    average = sum / len(arr)

    return average


def main():
    arr = [1,2,3,4]

    arrAverage = average(arr)

    print(f'Average of the arr is: {arrAverage}')


if __name__ == "__main__":
    main()

File: javascript/average.js
function average(...num){
    if(num.length <= 1){
        return num
    }

    let sum = 0

    for(let i of num){
        sum += i
    }

    return sum / num.length
}

function main(){
    average = average(1,2,3,4,5,6)

    console.log(`Average is: ${average}`)
}

main()